Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/ModelOverrider.cs

```diff
@@ -34,6 +34,8 @@ public class ModelOverrider : MonoBehaviour
         // The attached Agent
         Agent m_Agent;
 
+        private bool m_ProcessedCommandLine;
```

Comment:
Add a little comment explaining what this variable is and does.

Reply:
Done